packages/cli/test/mocks/project.ts

```diff
@@ -159,9 +159,23 @@ export function useProject(project: Partial<Project> = defaultProject) {
     res.json({ envs });
   });
   client.scenario.post(`/v8/projects/${project.id}/env`, (req, res) => {
-    envs.push(req.body);
+    const envObj = req.body;
+    envObj.id = envObj.key;
+    envs.push(envObj);
     res.json({ envs });
   });
+  client.scenario.delete(
+    `/v8/projects/${project.id}/env/:envId`,
+    (req, res) => {
+      const envId = req.params.envId;
+      for (const [i, env] of envs.entries()) {
+        if (env.key === envId) {
+          envs.splice(i, 1);
```

Comment:
```suggestion
          envs.splice(i, 1);
          break;
```

For correctness' sake, this should probably `break` here. If there were multiple env vars with the same `key` in this collection, this loop could produce unexpected behavior.